webpack.common.js

```diff
@@ -149,6 +150,7 @@ const CommonConfig = {
       patterns: [{ from: './app/src/public', to: './' }],
     }),
     new webpack.ProvidePlugin({
+      Buffer: ['buffer', 'Buffer'],
```

Comment:
This is to fix the following error after pbkdf2 upgrade:
Uncaught ReferenceError: Buffer is not defined

Reply:
Well that's annoying. Nice job finding the issue.